lldb/include/lldb/Target/Target.h

```diff
@@ -697,6 +698,37 @@ class Target : public std::enable_shared_from_this<Target>,
   Status Attach(ProcessAttachInfo &attach_info,
                 Stream *stream); // Optional stream to receive first stop info
 
+  /// Add or update a scripted frame provider descriptor for this target.
+  /// The descriptor's name is extracted from its scripted metadata class name.
```

Comment:
This should be a token returned from AddScriptedFrameProviderDescriptor, as we discussed above.